server/src/main/java/com/linecorp/centraldogma/server/CentralDogmaConfig.java

```diff
@@ -75,6 +81,44 @@
  */
 public final class CentralDogmaConfig {
 
+    private static final Logger logger = LoggerFactory.getLogger(CentralDogmaConfig.class);
+
+    private static final List<ConfigValueConverter> CONFIG_VALUE_CONVERTERS;
+
+    static {
+        final ArrayList<ConfigValueConverter> configValueConverters = new ArrayList<>();
+        Streams.stream(ServiceLoader.load(ConfigValueConverter.class)).forEach(configValueConverters::add);
+        configValueConverters.add(DefaultConfigValueConverter.INSTANCE);
+        CONFIG_VALUE_CONVERTERS = ImmutableList.copyOf(configValueConverters);
+        logger.debug("Available {}s: {}", ConfigValueConverter.class.getName(), CONFIG_VALUE_CONVERTERS);
+    }
+
+    @Nullable
+    static String convertValue(@Nullable String value) {
+        if (value == null) {
+            return null;
+        }
+
+        final int index = value.indexOf(':');
+        if (index < 0) {
+            // no prefix
+            return value;
+        }
+
+        final String prefix = value.substring(0, index);
```

Comment:
Should we early return if `prefix` is empty?

Reply:
> Do you mean a String that starts with :?

Right, I thought about that case.

Reply:
Updated in the above if condition. 😉